app/src/main/java/agency/five/codebase/android/movieapp/ui/component/ActorCard.kt

```diff
@@ -0,0 +1,86 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.model.Actor
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.AbsoluteRoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.scale
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+
+data class ActorCardViewState(
+    val imageUrl: String?,
+    val name: String,
+    val character: String,
+)
+
+@Composable
+fun ActorCard(
+    actorCardViewState: ActorCardViewState,
+    modifier: Modifier = Modifier,
+)
+{
+    Card(
+        modifier
+            .size(width = 150.dp, height = 270.dp),
+        elevation = 50.dp,
+        shape = AbsoluteRoundedCornerShape(18.dp),
+    ) {
+        Column {
+            Box(
+                modifier.size(200.dp)
+            ) {
+                AsyncImage(
+                    model = actorCardViewState.imageUrl,
+                    contentDescription = null,
+                    modifier
+                        .scale(1.35f)
+                        .size(170.dp)
+                )
+            }
+            Box(
+                modifier.background(color = Color.White)
+                    .size(width = 150.dp, height = 75.dp),
+                contentAlignment = Alignment.Center,
+            ) {
+                Column {
```

Comment:
You don't need to wrap this `Column` in a `Box`. You can just use column and pass it modifier which is currently passed to the box. Most often there is no need to wrap a single composable function in `Row`, `Column`,  `Box` etc.

Reply:
Fixed